supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAttributes.scala

```diff
@@ -90,4 +90,15 @@ object JesAttributes {
       case _ => "genomics.default-zones was set but no values were provided".invalidNel
     }
   }
+
+  def validateQps(config: Config): ErrorOr[PositiveInt] = {
+    val qpsCandidate = config.as[Option[Int]]("genomics-api-queries-per-100-seconds").getOrElse(GenomicsApiDefaultQps) / 100
+
+    if (qpsCandidate > 0) PositiveInt(qpsCandidate).validNel
+    else "Supplied QPS for Google Genomics API was not positive, value was {} using {} instead".invalidNel
+  }
+}
+
+final case class PositiveInt(value: Int) {
+  if (value < 0) throw new RuntimeException("PositiveInt cannot have a value below 0!")
```

Comment:
We had a convo already about my thoughts on PositiveInt. Either use something real or ditch the idea in general.